hasura/seeds/seedscript.ts

```diff
@@ -0,0 +1,152 @@
+import { writeFile } from "fs/promises";
+import { datatype, date, image, name, random } from "faker";
+// @ts-ignore
+const jsonSql = require("json-sql")();
+jsonSql.configure({
+  dialect: "postgresql",
+  wrappedIdentifiers: true,
+  separatedValues: false,
+});
+
+type Agency = { name: string };
+type Skill = { id: string; name: string; categoryId: string };
+
+const agencies: Agency[] = require("./Agency.json");
+const categories = require("./Category.json");
+const skills: Skill[] = require("./Skill.json");
+const topics = require("./Topic.json");
+
+type User = {
+  email: string;
+  name: string;
+  picture: string;
+};
+
+type UserAgency = {
+  userEmail: string;
+  agency: string;
+  created_at: string;
+};
+
+type UserSkill = {
+  userEmail: string;
+  skillId: string;
+  level: number;
+  created_at: string;
+};
+
+const config = {
+  nbUser: 500,
+  nbSkillEntriesPerUser: 30,
+  dateStart: "2020/01/01",
+  dateEnd: "2021/12/30",
+};
+
+const users: User[] = [];
+const userSkills: UserSkill[] = [];
+const userTechnicalAppetites: UserSkill[] = [];
```

Comment:
Isn't it called "desire" elsewhere?

Reply:
It's TechnicalAppetite in the database, desire sometimes on the app because the first one is a mouthful 😄